test/snapp_base.js

```diff
@@ -116,6 +116,8 @@ contract("SnappBase", async (accounts) => {
 
     it("Can't exceed max tokens", async () => {
       const instance = await SnappBase.new()
+
+      SnappBase.ca
```

Comment:
what is this? I have seen it several times :)

Reply:
Found it and removed it, please let me know if you see it anywhere else.